blog_spec.json

```diff
@@ -18,5 +18,15 @@
             "excludes_all": ["Load Shedding in Clojure", "Rastapopoulos"]
         },
         "twitter_username": "@mourjo_sen"
+    },
+    {
+        "link": "https://devblogs.microsoft.com/landingpage/",
+        "filter": {
+            "includes_any": [
+                "javascript"
```

Comment:
There seems to be no post that matches this filter. Please suggest a more relevant filter.

Reply:
Ah my bad, making a change, thanks